### DISCUSSION TITLE
show column description doesn't work

### DISCUSSION THREAD

**Discussion by youngynj19** [dbeaver/dbeaver#17953] [Q&A]
### Description

I set "show column description" on but cannot see them on a result panel.
The database connection is DB2 iSeries/AS 400.

How can I fix this?

### DBeaver Version

Community Edition 22.1.4

### Operating System

Windows 11

**Comment by LonwoLonwo** [comment]
Hello @youngynj19 

Can you provide your query example?

**Comment by youngynj19** [comment]
Hi @LonwoLonwo

I am afaraid I can't for a security matter... Is there another way to fix this without providing it?

thank you

**Comment by LonwoLonwo** [comment]
It can be related with this ticket: https://github.com/dbeaver/dbeaver/issues/14974

Do you have JOINs in your query?

**Comment by youngynj19** [comment]
oh, mine is just simple query. Like,

select * from table where column1=1 and column2=1

**Comment by LonwoLonwo** [comment]
Ok. Can you check this solution please? https://github.com/dbeaver/dbeaver/issues/14269#issuecomment-1046100904

**Comment by youngynj19** [comment]
wow, there is nothing written on the comment field..

![image](https://user-images.githubusercontent.com/97592790/194486413-c0c1a60f-388e-4b84-ac6a-1aead5d0f8fa.png)

Do you think this might be the reason why I can't see anything on the result panel?

**Comment by LonwoLonwo** [comment]
Do you have any comments there? 

Comments representation depends on some of driver properties https://github.com/dbeaver/dbeaver/issues/13153

**Comment by youngynj19** [comment]
Dear @LonwoLonwo.
You are my savior! #13153 solved the problem, thank you!!
